fix: edit support link (#421)
CleverCloud/documentation: content/doc/find-help/troubleshooting.md

GitHub does not give us access to organizations created or joined *after* you've linked your GitHub account to Clever Cloud (which is a good thing). So you need to let the Clever Cloud API access it. You can do that on <https://GitHub.com/settings/applications>.

> You can of course reach to [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice) if this page was not helpful enough.

Reviewer: ```suggestion
Feel free to each to [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice) if this page wasn't helpful enough.
```
